lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.Model.rb

```diff
@@ -590,4 +590,58 @@ def thermal_zone_prm_lab_delta_t(thermal_zone)
   def model_get_fan_power_breakdown
     return true
   end
+
+  def model_set_spm(model, thermalZones, coil)
```

Comment:
We need to add YARD comments for this methods. I would also rename it to something more specific, `model_set_spm` is too broad, maybe something like `model_set_central_preheat_coil_spm()`?